src/components/MailingList/MailingList.test.jsx

```diff
@@ -1,8 +1,17 @@
-import axios from "axios";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MailingList, { addEmailFetch } from './MailingList';
+import { mount } from 'enzyme';
+import { Get } from '../../utility/api';
```

Comment:
it seems fairly inefficient that in both our calendar and mailing list tests, we have the same or more lines of imports as actual tests. Is it worth looking at consolidation for now since we have fairly minimal testing, or do we want to keep these all in separate files?

Reply:
It is an option to throw everything into a `__tests__` folder, but I still think separate files is better organization for BDD unit tests, and the extra LOC don't bother me much.